src/styles/nav-bar.scss

```diff
@@ -0,0 +1,104 @@
+.NavBar {
+    background-color: lightskyblue;
+    color:white;
+
+    @media (max-width: 600px) {
+        display: none;
+    }
+    
+    &__ul {
+        @media (max-width: 600px) {
+            list-style: none;
+            display: flex;
+            margin: 5px 0px;
+            padding: 0px;
+            flex-direction: column;
+
+        }
+
+        @media (min-width: 601px) {
```

Comment:
There is no need to add 2 media queries here, standard approach is mobile-first so you add styles that will be applied normally and then modify them with query filter with`min-width:  600px`